linux/Platform.c

```diff
@@ -249,6 +251,13 @@ void Platform_setSwapValues(Meter* this) {
    this->values[0] = pl->usedSwap;
 }
 
+void Platform_setZramValues(Meter* this){
+   LinuxProcessList* lpl = (LinuxProcessList*) this->pl;
```

Comment:
Please use `const LinuxProcessList` to avoid conflicts with #274

(`lpl` is only read from)